dev-docs/feature-conditional.md

```diff
@@ -0,0 +1,239 @@
+            }
+       }
+   ]
+}]
+
+{% endhighlight %}
+
+Example of how this works:
+
+1. Users with a screen width of 1000px will cause the second mediaQuery to fire.
+1. This rule puts "tablet" into the label array and a list of sizes into sizesSupported.
+1. Then the AdUnit is processed:
+    1. The first bid requires that the label "display" be present in the array. It's not, so that bid is skipped.
+    1. The second bid requires that either "phone" or "tablet" be present. Since tablet is in the label array, that bid is activated and the correct placement is sent to bidderA.
+
+## Use Case: What if some AdUnit auctions should be skipped entirely for some devices?
+
+Say there's a responsive page where one of the AdUnits only supports larger sizes, so it doesn't make sense
+on phones. We really don't want to use that AdUnit for mobile users. Wouldn't it be great if `labelAny` and `labelAll` could be applied to the entire AdUnit? They can. Here's an example:
+
+{% highlight js %}
+
+var AdUnits = [{
+    code: "ad-slot-1",
+    sizes: [[768,90]], 
+    labelAny: ["display", "tablet"], // skip the ad unit entirely for phones
+    bids: [
+        {
+            bidder: "bidderA",  // no conditions
+            params: {
+                placement: "1111"
+            }
+       },{
+            bidder: "tabletBidder",
+            labelAny: ["tablet"], // additional conditions can be applied to bids
+            params: {
+                placement: "2222"
+            }
+       }
+   ]
+}]
+
+{% endhighlight %}
+
+## Use Case: What if some bid requests apply only to users originating certain countries? 
+
+Labels aren't constrained to describing device size -- they can be used for any condition the page wants
+to define. To support this, labels can now be passed into the `requestBids()` function as an argument.
+
+A specific use case: suppose that a certain bidder doesn't have a data center outside of a
+certain region. It's really not worth sending them bid
+requests for users outside of their geographic area. Assuming the page can figure out where the user's from,
+a label can be implemented and applied to make the bid conditional.
+
+{% highlight js %}
+// page logic determines the 'europeanUser' boolean
+If (europeanUser) {
+    reqArgs={labels:['eur']};
+} 
+requestBids(reqArgs);
+{% endhighlight %}
+
+Then this label can be applied to conditions in the AdUnit just like labels that originate from sizeConfig. E.g.
+
+{% highlight js %}
+var AdUnits = [{
+    code: "ad-slot-1",
+    sizes: [[768,90], [468,60], [320,50]], 
+    bids: [
+       {
+            bidder: "euroMobileBidder",
+            labelAll: ["eur", "phone"],
+            params: {
+                placement: "9876"
+            }
+       },
+       ...
+   ]
+}]
+{% endhighlight %}
+
+This example shows that the 'euroMobileBidder' is only interested in receiving bids that have **both**
+labels: both "eur" as passed into requestBids and "phone" as created by sizeConfig.
```

Comment:
We should remove the second use of "both".  Alternatively, this info would be easier to consume in an unordered list, e.g.,

<blockquote>
that have both labels:
- eur as ...
- phone as ...
</blockquote>